src/reactionsystem.jl

```diff
@@ -694,77 +712,125 @@ function ReactionSystem(rxs::Vector, iv = Catalyst.DEFAULT_IV; kwargs...)
 end
 
 # search the symbolic expression for parameters or unknowns
-# and save in ps and sts respectively. vars is used to cache results
-function findvars!(ps, sts, exprtosearch, ivs, vars)
+# and save in ps and us respectively. vars is used to cache results
+function findvars!(ps, us, exprtosearch, ivs, vars)
```

Comment:
Quite a few places we used e.g. `sts` assuming that something is a state. When these was renamed to unknowns I also renamed these variables to make that clear.